hackathon_site/hardware/test_api.py

```diff
@@ -1631,6 +1677,56 @@ def test_no_remaining_quantities(self):
         }
         self.assertEqual(response.json(), expected_response)
 
+    def create_order(self):
```

Comment:
could you move this helper function to the top of the class, under create_min_number_of_profiles, we try to keep them near the top so they're all in one place